backend/settings_base.py

```diff
@@ -207,8 +208,14 @@
 # Number of seconds to count down
 CELERY_DELAY = 0
 
-DEFAULT_FROM_EMAIL = 'webmaster@localhost'
+# DEFAULT_FROM_EMAIL = 'webmaster@localhost'
 
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
```

Comment:
These are already setup in settings.py
E-Mailing is already setup for alumni login. Please check that out and use that